### ISSUE TITLE
Force Replace option not functioning due to typo.

### ISSUE THREAD

**Issue by seasnak** [gurugaurav/bing_image_downloader#23]
On line 24 in downloader.py there is a missing '_'.

`if Path.isdir(image_dir):` should be `if Path.is_dir(image_dir):`

**Comment by korreckj328** [comment]
Put in a pull request to fix this as I had the same issue with the typo.

**Comment by Seasnak** [comment]
> Put in a pull request to fix this as I had the same issue with the typo.

I created a PR when I found the issue, but it hasn't been approved/merged yet.

**Comment by korreckj328** [comment]
> > Put in a pull request to fix this as I had the same issue with the typo.
> 
> I created a PR when I found the issue, but it hasn't been approved/merged yet.

Yeah I realized after a little more looking that this repo is dead and I just forked it to start improving it myself for what I'm working on.  First time looking at scraping images to make a classifier and I am kind of shocked how little seems to be out there for this task.
